### PR TITLE
1123 - updated conference link and explanation

### LINKED ISSUE #1123: Bug: README link for conference points to pharmaverse repo now
### What happened?

Explain the US connect 2022 workshop in the readme, which also contains admiral slides from EU 2021.

### Session Information

NA

### Reproducible Example

NA

### PR DESCRIPTION
**Pull request by rossfarrugia**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

Closes #1123

### CODE REVIEW

**Comment by rossfarrugia** [comment]
@aehmann-gsk i put you down as reviewer given the discussion over email for this minor update. i've asked @thomas-neitmann to double check the automated check fails.

File: README.md [hunk 1]
```diff
@@ -136,7 +136,8 @@ therefore want any module to be understandable to all, not only the `{admiral}`
 ## Conference Presentations
 
 * [R/Pharma 2021 talk](https://www.youtube.com/watch?v=N7Bw8c3D5fU) (recording)
-* [PHUSE EU Connect 2021 workshop](https://github.com/pharmaverse/admiral.phuse.workshop/tree/main/EU%202021) (slides and materials)
+* [PHUSE US Connect 2022 pharmaverse workshop](https://github.com/pharmaverse/pharmaverse.workshop.phuseUS2022) (slides and materials - including `{admiral}`
+workshop slides from PHUSE EU Connect 2021)
```
Comment (aehmann-gsk) [inline README.md:140]: Change to PHUSE US Connect 2022

Comment (aehmann-gsk) [reply README.md:140]: Line 140

Comment (rossfarrugia) [reply README.md:140]: that section was to highlight that we also have the `admiral` EU slides there in addition to all the materials from the US `pharmaverse` workshop

**Review by aehmann-gsk — APPROVED** [review]

**Comment by rossfarrugia** [comment]
@thomas-neitmann i'm just going to merge this one with admin privilege as its only a one line update in readme

**Pull request merged**